hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcClient.java

```diff
@@ -1580,7 +1580,7 @@ private void acquirePermit(final String nsId, final UserGroupInformation ugi,
         // Throw StandByException,
         // Clients could fail over and try another router.
         if (rpcMonitor != null) {
-          rpcMonitor.getRPCMetrics().incrProxyOpPermitRejected();
```

Comment:
Should we keep the old method too?

Reply:
@goiri ,this one line is kept and moved to FederationRPCPerformanceMonitor.java line 190